doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -482,7 +483,25 @@ If the configuration is successfully downloaded, the system asks for the followi
 * Security server code (**reference data: 2.4**), which is chosen by the security server administrator and which has to be unique across all the security servers belonging to the same X-Road member.
 * Software token’s PIN (**reference data: 2.5**). The PIN will be used to protect the keys stored in the software token. The PIN must be stored in a secure place, because it will be no longer possible to use or recover the private keys in the token once the PIN has been lost.
 
-### 3.4 Configuring configuration backup encryption
+### 3.4 Configuring firewall
+
+It is strongly recommended to protect the security server from unwanted access using a firewall (hardware or software based). The firewall can be applied to both incoming and outgoing connections depending on the security requirements of the environment where the security server is deployed. 
+
+**Special attention should be paid with the firewall configuration since incorrect configuration may leave the security server vulnerable to exploits and attacks.** This type of abuse could result in compromised access to the Security Server and the data that is exchanged through it.
+
+It is recommended to allow incoming traffic to specific ports only from explicitly defined sources using IP filtering. Access for ports 80, 443 and 4000 should be especially defined, as these ports are used for making X-Road queries and accessing the user interface.
```

Comment:
We're about to change the client information system default communication ports from 80/443 to 8080/8443 on Ubuntu. Therefore, I would either use the new default ports or mention both old and new default ports (80/8080, 443/8443).

Reply:
Updated